core/templates/dev/head/pages/review-test-page/review-test-page.mainpage.html

```diff
@@ -30,7 +30,7 @@
   <script src="/third_party/static/ckeditor-4.9.2/ckeditor.js"></script>
 
   <script src="/templates/dev/head/mathjaxConfig.js"></script>
-  <script src="/third_party/static/MathJax-2.6.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
+  <script src="/third_party/static/MathJax-2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
```

Comment:
Is this an intended change?

Reply:
Yes I am keeping it consistent with the practice sessions import